packages/server/test/unit/plugins/child/run_child_fixture.js

```diff
@@ -7,7 +7,7 @@ let proc
 
 process.on('message', (msg) => {
   if (msg.msg === 'spawn') {
-    proc = childProcess.fork(REQUIRE_ASYNC_CHILD_PATH, ['--projectRoot', msg.data.projectRoot, '--file', path.join(msg.data.projectRoot, 'cypress.config.js')])
+    proc = childProcess.fork(REQUIRE_ASYNC_CHILD_PATH, ['--projectRoot', msg.data.projectRoot, '--configFilePath', path.join(msg.data.projectRoot, 'cypress.config.js')])
```

Comment:
as a pendant to the other comment, I would prefer to keep the word `file` here as well.

Reply:
https://github.com/cypress-io/cypress/pull/20560/commits/7bcae3aa2e90d71e350abf67b385ff1bcea3883f